package j$.util.stream;

/* loaded from: classes2.dex */
public final /* synthetic */ class U implements j$.util.function.E {
    public final /* synthetic */ int a;
    public final /* synthetic */ e2 b;

    public /* synthetic */ U(int i, e2 e2Var) {
        this.a = i;
        this.b = e2Var;
    }

    @Override // j$.util.function.E
    public final void accept(int i) {
        switch (this.a) {
            case 0:
                this.b.accept(i);
                break;
            default:
                ((W) this.b).a.accept(i);
                break;
        }
    }

    @Override // j$.util.function.E
    public final /* synthetic */ j$.util.function.E l(j$.util.function.E e) {
        switch (this.a) {
        }
        return j$.com.android.tools.r8.a.c(this, e);
    }
}
